dashboard/config/initializers/backtrace_silencers.rb

```diff
@@ -8,8 +8,9 @@
 
 # In addition to backtrace silencing, we also want to silence annoying deprecations:
 silenced = [
-  # Added in https://github.com/rails/rails/pull/25337 for Rails 5.1; will be removed in Rails 5.2
-  /The behavior of `[\w\?]*` inside of after callbacks will be changing in the next version of Rails./,
+  /Single arity template handlers are deprecated/,
```

Comment:
Is there a way to only suppress these when `RAILS_ENV` is `production`? I imagine we should be looking closely at some of these in `development` and `test` to prepare for future Rails upgrades?

Reply:
Yep, I'm actively planning on addressing each of these as part of the work I'm doing to upgrade to Rails 6. I think now that we are aware of the deprecations and have them written down somewhere - specifically, here - I'm not worried about preserving the warnings themselves.